validator/publication.py

```diff
@@ -1,27 +1,19 @@
-import requests
 from validator.generic import GenericValidator
+from validator.request.connector import Connector, NotFound
+
 
 class Publication():
 
     def __init__(self, doi, pubmed_id):
         self.doi = doi
         self.PMID = pubmed_id
 
-    def populate_from_eupmc(self):
-        payload = {'format' : 'json'}
-        eupmc_url = 'https://www.ebi.ac.uk/europepmc/webservices/rest/search'
+    def populate_from_eupmc(self, connector: Connector):
```

Comment:
Same here with `DefaultConnector` there instead of the abstract `Connector` ?

Reply:
Ditto